scripts/auto_upgrade_service.py

```diff
@@ -22,6 +22,13 @@
         service_id = os.environ.get('RANCHER_TARGET_SERVICE_ID')
         print 'Service ID: {0}'.format(service_id)
         service = api.services(service_id)
+
+        # Use the new image version
```

Comment:
```suggestion
        # Use the new image version
        # Replace image being used in service RANCHER_TARGET_SERVICE_ID
        # into RANCHER_NEW_IMAGE_UUID
        # Useful for upgrading new releases.
```

Reply:
LGTM @dimasciput . Let's try suggestion Beta and merge this with this new comments.